app/components/course_details/view.rb

```diff
@@ -30,7 +31,7 @@ def rows
           collection.unshift({
             key: t("components.course_detail.course_details"),
             value: course_details,
-            action_href: edit_trainee_publish_course_details_path(trainee),
+            action_href: edit_trainee_course_education_phase_path(trainee),
```

Comment:
Is this definitely the right place for this link? I would expect the "Course details" change button to take me to the `publish-course-details/edit` page

Reply:
Yes, I think you're right. I might have accidently change it here while figuring out the flow.

Reply:
I think @archferns is right